logger.js

```diff
@@ -9,7 +9,7 @@ function logger (opts, stream) {
   function pino (ctx, next) {
     wrap(ctx.req, ctx.res)
     ctx.log = ctx.request.log = ctx.response.log = ctx.req.log
-    ctx.onerror = catchErr(ctx, ctx.onerror)
+    ctx.onerror = catchErr(ctx, ctx.onerror.bind(ctx))
```

Comment:
I don't think the v8 which has `bind` optimized has landed in v8 yet, we should instead do `catchErr(ctx, function () { return ctx.onerror.apply(ctx, arguments) }))`

`bind` (before the it was optimized in later versions of v8) has notoriously bad performance